src/Symfony/Component/PropertyAccess/PropertyAccessor.php

```diff
@@ -271,11 +271,19 @@ private function readPropertiesUntil(array $zval, PropertyPathInterface $propert
         // Add the root object to the list
         $propertyValues = [$zval];
 
-        for ($i = 0; $i < $lastIndex; ++$i) {
+        for ($i = $startIndex; $i < $lastIndex; ++$i) {
             $property = $propertyPath->getElement($i);
             $isIndex = $propertyPath->isIndex($i);
             $isNullSafe = $propertyPath->isNullSafe($i);
 
+            $isWildcard = false;
+            if (method_exists($propertyPath, 'isWildcard')) {
+                // To be removed in symfony 8 once we are sure isNullSafe is always implemented.
```

Comment:
Should `isNullSafe` be replaced with `isWildcard`?

Reply:
@slimus oh no, i copied that comment and forgot to put `isWildcard ` there

Reply:
https://github.com/symfony/symfony/pull/52723/commits/16c1c1cd2b5d649bbe4c44d1f0572edcc8203f42 fixed here